examples/browser/package.json

```diff
@@ -39,6 +40,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
+    "bunyan": "^1.8.10",
```

Comment:
I see, you already have `bunyan` as dependencies in `theia-core`, so keeping it here as `dependencies` should be OK